user/src/main/java/org/tessell/widgets/RowTable.java

```diff
@@ -62,6 +62,18 @@ public void addRow(final IsWidget isWidget) {
     addRow(isWidget.asWidget());
   }
 
+  /** Assumes {@code row} is a table row and returns its index */
+  @Override
+  public int indexOfRow(final Widget row) {
+    return rows.indexOf(row);
+  }
+
+  /** Assumes {@code row} is a table row and returns its index */
+  @Override
+  public int indexOfRow(final org.tessell.gwt.user.client.ui.IsWidget row) {
+    return rows.indexOf(row);
```

Comment:
If you look at where "rows" is maintained, the contents (in this class) are always the real Widgets, so you should call row.asWidget() here.